fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlTranslator.java

```diff
@@ -163,4 +165,32 @@ public String getUrl(Properties connectionProperties) {
     public boolean clobSupportsInline() {
         return false;
     }
+    
+    /* (non-Javadoc)
+     * @see com.ibm.fhir.database.utils.api.IDatabaseTranslator#getType()
+     */
+    @Override
+    public DbType getType() {
+        return DbType.POSTGRESQL;
+    }
+
+    /* (non-Javadoc)
+     * @see com.ibm.fhir.database.utils.api.IDatabaseTranslator#dualTableName()
+     */
+    @Override
+    public String dualTableName() {
+        // PostgreSQL does not support a "DUAL" table because the FROM clause is optional.
+        return null;
```

Comment:
it's kind of messy to return null, I realize there is overhead for this, but return empty I think is preferred.

Reply:
I find null here a useful flag to indicate no such value exists for this implementation. This should be used consistently, to distinguish between no value and an actual value which happens to be an empty string (something which databases don't handle consistently).

Reply:
Right... it's actually just SELECT 1 in postgres.   I think the concept shouldn't be so much focused on a 'dualTableName' rather connectivityCheck.

Reply:
I'm leaving this as resolved